code/online_challenges/src/hackerrank/absolute_permutation/absolute_permutation.cpp

```diff
@@ -0,0 +1,29 @@
+#include <bits/stdc++.h>
```

Comment:
#1915.

Reply:
Changes done, please review

Reply:
> #1915.

Sir, the issues have been resolved, please review!